test/EPiServer.Marketing.Testing.Test/Web/KpiWebRepositoryTests.cs

```diff
@@ -129,7 +129,7 @@ public void GetKpiInstance_calls_KpiManager_Get()
             Guid testGuid = Guid.Parse("48fe0e90-67f9-4171-a65c-aa00efc0ce77");
             var webRepo = GetUnitUnderTest();
             webRepo.GetKpiInstance(testGuid);
-            _mockKpiManager.Verify(call => call.Get(It.Is<Guid>(val => val == testGuid)),Times.Once);
+            _mockKpiManager.Verify(call => call.Get(It.Is<Guid>(val => val == testGuid)), Times.Once);
```

Comment:
I don't know why, but this one struck me as the funniest change ever.

Reply:
This was the last step that was needed before this code is ready to be released to the world!